src/battle-tooltips.ts

```diff
@@ -997,6 +997,8 @@ class BattleTooltips {
 
 			if (this.battle.gen > 2 && ability === 'quickfeet') {
 				stats.spe = Math.floor(stats.spe * 1.5);
+			} else if (this.battle.gen < 2 && pokemon.status === 'par') {
```

Comment:
Isn't this excluding Gen 2? I realize the burn check above does the same thing but that seems wrong too.

Reply:
The check above excludes Gen 1 and 2, but that should be fine since they don't have Abilities anyway.